openassessment/templates/openassessmentblock/edit/oa_edit_basic_settings_list.html

```diff
@@ -161,8 +161,10 @@
                     <label for="openassessment_teamset_selector" class="setting-label">{% trans "Select Team Set"%}</label>
                     <select id="openassessment_teamset_selector" class="input setting-input" name="teamset selection">
                         <!-- TODO: provide real team names to select from -->
-                        <option value="team-a" selected="true">{% trans "Team A"%}</option>
-                        <option value="team-b">{% trans "Team B"%}</option>
+                        {% for team in team_names %}
```

Comment:
Nit: consider renaming the `team_names` context variable and the `team` iteration variable - these are really team-set names, not team names.